includes/html/pages/services.inc.php

```diff
@@ -236,13 +235,8 @@
                         echo '</tr>';
 
                         if ($service_iteration >= $services_count) {
-                            $footer = true;
```

Comment:
You left the $footer initialization up above.

Reply:
OK, $footer initialization removed